Generator/generator-botbuilder-python/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "generator-botbuilder-python",
+  "version": "0.0.0",
+  "description": "Template to create conversational bots in Python using Microsoft Bot Framework.",
+  "homepage": "https://github.com/Microsoft/botbuilder-python",
+  "author": {
+    "name": "Microsoft",
+    "email": "v-frgome@microsoft.com",
```

Comment:
This doesn't seem right, if the email field is being provided, this should be going to a general support alias. (Not that I know what it is, it might be [_botframework@microsoft.com_](https://github.com/Microsoft/BotBuilder/blob/cc35ac5af30de5898514f53a509538df232952dd/CSharp/Library/Microsoft.Bot.Connector.Shared/Swagger/ConnectorAPI.json#L11))

If you can find an email to use, great, otherwise this should be empty like in your [package.json](https://github.com/Microsoft/botbuilder-js/blob/master/Generator/generator-botbuilder-js/package.json#L8) for botbuilder-js.

Reply:
Oops! I thought I got rid of this...